packages/console/src/components/UnsavedChangesAlertModal/index.tsx

```diff
@@ -27,10 +34,13 @@ const UnsavedChangesAlertModal = ({ hasUnsavedChanges }: Props) => {
       return;
     }
 
-    const {
-      block,
-      location: { pathname },
-    } = navigator as BlockerNavigator;
+    /**
+     * Props `block` and `location` are removed from `Navigator` type in react-router, for the same reason as above.
+     * So we have to define our own type `BlockerNavigator` to acquire these props that actually exist in `navigator` object.
+     */
+    // eslint-disable-next-line no-restricted-syntax
+    const { block, location } = navigator as BlockerNavigator;
+    const { pathname } = location;
```

Comment:
Nothing in particular. Just simply looks better. 😂
Otherwise it would be something like this:

```ts
const {
  block,
  location: { pathname },
  // eslint-disable-next-line no-restricted-syntax
} = navigator as BlockerNavigator;
```

Reply:
lol